sysutils/backup-station/pkg-plist

```diff
@@ -1,4 +1,10 @@
 bin/backup-station
 etc/sudoers.d/backup-station
+%%PYTHON_SITELIBDIR%%/__pycache__/setup%%PYTHON_EXT_SUFFIX%%.opt-1.pyc
+%%PYTHON_SITELIBDIR%%/__pycache__/setup%%PYTHON_EXT_SUFFIX%%.pyc
+%%PYTHON_SITELIBDIR%%/build/lib/__pycache__/setup%%PYTHON_EXT_SUFFIX%%.opt-1.pyc
+%%PYTHON_SITELIBDIR%%/build/lib/__pycache__/setup%%PYTHON_EXT_SUFFIX%%.pyc
+%%PYTHON_SITELIBDIR%%/build/lib/setup.py
+%%PYTHON_SITELIBDIR%%/setup.py
```

Comment:
All %%PYTHON_SITELIBDIR%% lines should be removed if not needed.

Reply:
Thanks,  they should be removed.